transport/conn.go

```diff
@@ -91,12 +91,18 @@ type connReader struct {
 
 	h  map[frame.StreamID]responseHandler
 	mu sync.Mutex
+
```

Comment:
Let's move stream up to group 2, you could actually rename it to `s`.
It should be noted that mu protects both `h` and `s`.